benchmark/src/grpc_helpers.rs

```diff
@@ -92,25 +92,65 @@ fn check_ac_response(resp: &ProtoSubmitTransactionResponse) -> bool {
 /// Ignore but count both gRPC-failed submissions and AC-rejected TXNs.
 pub fn submit_and_wait_txn_requests(
```

Comment:
it's not only for txn_requests now, let's update the function name

Reply:
There are a few methods that refer to txn_requset / txn_load -- these should be updated, since reads are not really transaction requests.

Reply:
Please see my general response about renaming.